cmd/grafaind/scenario_test.go

```diff
@@ -178,11 +189,51 @@ func initGenesis(t *testing.T, filename string, alice weave.Address) {
 			"cash": cash.Configuration{
 				CollectorAddress: alice,
 			},
+			"msgfee": dict{
+				"owner":     alice,
+				"fee_admin": alice,
+			},
+		},
+		"rbac": dict{
+			"roles": []dict{
+				{
+					"name":  "system.admin",
+					"owner": "seq:rbac/role/1",
+					"permissions": []string{
+						"_grafain.artifacts.delete",
+					},
+				},
+				{
+					"name":     "k8s.devops",
+					"owner":    "seq:rbac/role/1",
+					"role_ids": []int{1},
+					"permissions": []string{
+						"_grafain.artifacts.create",
+					},
+				},
+			},
+			"principals": []dict{
```

Comment:
This is the only part I find somewhat confusion... using these principals instead of addresses / conditions. I guess to give them a consistent name? 

Using addresses that ref multisig will give the same visiblity (if we add a name field to multisig). `seq:multisig/contract/2` then look that up and see a name and list of included signatures (possibly embedded multisigs).

Is there some other utility to using principals over plain addresses that I miss?

Reply:
yes, the main reason for a principal model is to enrich it with metadata. This can either be human or service meta data.